### ISSUE TITLE
A11Y issue - Wrong semanticLabel and semanticHint for disabled MaterialPinField

### ISSUE THREAD

**Issue by epentesi** [adar2378/pin_code_fields#427]
## Summary

`MaterialPinField` exposes an `enabled` parameter that allows users to disable the widget when `enabled: false`. However the screen readers continue to read the default hardcoded semanticLabel *"6-digit PIN code field"* and then *"**Text field is editing**. Insertion point at start"* and the hardcoded semanticHint *"Enter 6 more digits"*.

## Current behavior

For a visually impaired user it seems that the field can be edited but it is not true.

## Expected behavior

The screen reader should be able to detect the non-editability of the PIN code field and to read aloud something like "Text field, dimmed" (as for a CONFIRM button _"Confirm, **dimmed**, button"_) automatically (and not hardcoded) in the language set in the OS.

**Comment by adar2378** [comment]
Thanks for the report, will check it buddy!
